src/DashboardView/Settings/Settings.scss

```diff
@@ -0,0 +1,8 @@
+@import "EventTypes/EventTypes";
+.card-header h5 a {
+  color: $COLOR-font-dark !important;
```

Comment:
Is !important really necessary here? Can't you just use a class selector or anything more specific?